docs/04-operation-guides/operations/obsv-04-enable-mtls-for-custom-metrics.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Enable mTLS for custom metrics    
+---
+
+To enable Strict mTLS scraping for a component, configure the Istio TLS certificates in the corresponding ServiceMonitor:
```

Comment:
```suggestion
To enable Strict mTLS scraping for a component, configure the Istio TLS certificates in the corresponding Service Monitor:
```

Reply:
Didn't we agree to use natural language? 🤔

Reply:
Totally! 😅